sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGetJsonObject.scala

```diff
@@ -32,9 +106,23 @@ case class GpuGetJsonObject(json: Expression, path: Expression)
   override def nullable: Boolean = true
   override def prettyName: String = "get_json_object"
 
-  override def doColumnar(lhs: GpuColumnVector, rhs: GpuScalar): ColumnVector = {    
-    lhs.getBase().getJSONObject(rhs.getBase, 
-    GetJsonObjectOptions.builder().allowSingleQuotes(true).build());
+  def normalizeJsonPath(path: GpuScalar): Option[Scalar] = {
```

Comment:
The way this code is written it looks like it is going to leak the Scalar.  Can we make this an `Option[String]` instead? That would also let us cache it, because we know that it is a constant value.

Reply:
Thanks, done